backend/game/cards/keep_cards/victory_point_manipulation_cards/test_urbavore.py

```diff
@@ -0,0 +1,19 @@
+from game.cards.keep_cards.victory_point_manipulation_cards.urbavore import Urbavore
+from game.values import locations
+
+
+def test_urbavore_player_add_card(player):
+    Urbavore().immediate_effect(player, None)
+    assert player.card_on_hand.get("Urbavore")
+
+
+def test_urbavore_costs_4_energy():
+    assert Urbavore().cost == 4
+
+
+def test_urbavore_gain_2_star(player):
+    player.victory_points = 4
```

Comment:
Why?

Reply:
Removed the preset, was looking to better show that it added 1 to 4 as opposed to default of adding 1 to 0